managed-ledger/src/test/java/org/apache/bookkeeper/test/ZooKeeperUtil.java

```diff
@@ -116,7 +116,7 @@ public void startServer(String path) throws Exception {
         // create a zookeeper client
         LOG.debug("Instantiate ZK Client");
         zkc = ZooKeeperClient.newBuilder().connectString(getZooKeeperConnectString()).build();
-        if (path != "") {
+        if (StringUtils.isNotEmpty(path)) {
```

Comment:
`if (!path.isEmpty()) ` is enough here, no need to use a third party library

Reply:
In fact, I changed to `StringUtils.isNotEmpty` because of @michaeljmarshall  review.
https://github.com/apache/pulsar/pull/11497#discussion_r684524851
In my opinion, we should not use unnecessary third-party libraries. I think `! "".equals(path)` is also ok because `NullPointerException` can be avoided, thanks

Reply:
Sorry for the back and forth here @skyguard1. I only recommended using `StringUtils.isNotEmpty` because it is used elsewhere in the project. `! "".equals(path)` certainly works, too.

Reply:
Don't care, it's not a big thing